package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lint-staged",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
```

Comment:
This might be a breaking change, requiring `npm@7` while Node.js 14 ships with `npm@6` by default...

Reply:
Never mind, this isn't published to `npm` so it only affects development of _lint-staged_ itself.